archivebox/core/admin.py

```diff
@@ -66,6 +66,12 @@ class SnapshotAdmin(admin.ModelAdmin):
     actions = [delete_snapshots, overwrite_snapshots, update_snapshots, update_titles, verify_snapshots]
     actions_template = 'admin/actions_as_select.html'
 
+    def get_queryset(self, request):
+        return super().get_queryset(request).prefetch_related('tags')
+
+    def tag_list(self, obj):
+        return u", ".join(o.name for o in obj.tags.all())
```

Comment:
```suggestion
        return ', '.join(obj.tags.values_list('name', flat=True))
```

(In python 3 all strings are `u""` strings by default.)